configs/fcos/fcos_r50_caffe_fpn_4x4_1x_coco.py

```diff
@@ -12,7 +12,7 @@
         num_stages=4,
         out_indices=(0, 1, 2, 3),
         frozen_stages=1,
-        norm_cfg=dict(type='BN', requires_grad=False),
+        norm_cfg=dict(type='BN', requires_grad=True),
```

Comment:
Why do you modify this?

Reply:
The `samples_per_gpu` and `workers_per_gpu` should change to 2 now.

Reply:
Recently, I have read the mmdetection paper and noticed that the requires_grad=True is the default setting in mmdetection. I thought that this requirese_grad being set as False for FCOS only was due to the async num_pos issue. So I changed requires_grad=True after fixing the num_pos bug.

**But**, after running an initial experiment, I found that setting requires_grad=True hurt the performance for over 1 mAP.    I am running another experiment with requires=False to see the performance. If the performance is good, that means setting requires_grad=True is not a good choice and I will change it back to False. Otherwise, I need to debug the code further.

Reply:
It depends on the pretrained backbone you use. In the paper we use pytorch style backbones and pretrained weights from torchvision. However, `open-mmlab://detectron/resnet50_caffe` is not a normal pretrained weight, which fused conv and bn layers and the weight/bias of bn layers are 0 and 1. Thus we should not set `requires_grad=True`.

Reply:
That makes sense and my experiment results also agree with this. I will change it back and do some minor changes to the over-length lines.